packages/graphql/lib/src/links/websocket_link/websocket_client.dart

```diff
@@ -86,30 +87,43 @@ class SocketClientConfig {
   /// Warning: if you want to listen to the listen to the stream,
   /// wrap your channel with our [GraphQLWebSocketChannel] using the `.forGraphQL()` helper:
   /// ```dart
-  /// connect: (url, protocols) {
+  /// customConnect: (url, protocols) {
```

Comment:
Do you think we should be consistent with the naming of this and the `fnConnect` function? That way, people will know that the functions basically do the same.

Reply:
I agree, I will move all under the name `connectFn`